src/client.ts

```diff
@@ -55,8 +55,32 @@ export default class UnleashClient extends EventEmitter {
     }
   }
 
+  isParentDependencySatisfied(feature: FeatureInterface | undefined, context: Context) {
+    if (!feature?.name || !feature.dependencies?.length) {
```

Comment:
What's with the name check here? Name should always be a non empty string, do I have the dumb or is the same as 

if (!feature?.dependencies?.length) {
  return true;
}

Reply:
still passes the spec. not sure why I did the name check but it's not in the type anymore. I'm hallucinating :D

Reply:
Removed